src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInStudentProfileAttributes.java

```diff
@@ -0,0 +1,110 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.List;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.client.scripts.util.DataMigrationForSanitizedDataHelper;
+import teammates.client.scripts.util.LoopHelper;
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.logic.core.ProfilesLogic;
+
+/**
+ * Script to desanitize content of {@link StudentProfileAttributes} if it is sanitized.
+ *
+ * <p>Fields {@link StudentProfileAttributes#shortName}, {@link StudentProfileAttributes#email},
+ * {@link StudentProfileAttributes#institute}, {@link StudentProfileAttributes#nationality},
+ * {@link StudentProfileAttributes#gender}, {@link StudentProfileAttributes#moreInfo}
+ * are no longer sanitized before saving and these fields are expected to be in their unsanitized form.</p>
+ *
+ * <p>This script desanitizes these fields of exisiting StudentProfileAttributes if they are sanitized so that
+ * all profiles will have unsanitized values in these fields.</p>
+ */
+public class DataMigrationForSanitizedDataInStudentProfileAttributes extends RemoteApiClient {
+
+    // will not perform updates on the datastore if true.
+    private static final boolean isPreview = true;
+    private ProfilesLogic profilesLogic = ProfilesLogic.inst();
+
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForSanitizedDataInStudentProfileAttributes().doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        List<StudentProfileAttributes> allStudentProfiles = profilesLogic.getAllStudentProfiles();
+        int numberOfAffectedProfiles = 0;
+        int numberOfUpdatedProfiles = 0;
+        LoopHelper loopHelper = new LoopHelper(100, "student profiles processed.");
+        println("Running data migration for sanitization on student profiles...");
+        println("Preview: " + isPreview);
+        for (StudentProfileAttributes profile : allStudentProfiles) {
+            loopHelper.recordLoop();
+            boolean hasAllUnsanitizedFields = !hasAnySanitizedField(profile);
+            if (hasAllUnsanitizedFields) {
+                // skip the update if all fields are already unsanitized
+                continue;
+            }
+            // profile contains sanitized fields
+            numberOfAffectedProfiles++;
```

Comment:
There are two ways to attach a comment to a specific line. In the second case, we need a blank line after the statement (to avoid attaching it to the whole block)
```java
statement1; // comment

// comment
statement1;

statement2;
statement3;
```